.github/workflows/documentation.yaml

```diff
@@ -0,0 +1,70 @@
+name: Documentation
+
+on:
+  push:
+    branches:
+    - master
+#  pull_request:
+#    branches:
+#    - master
+
+jobs:
+  github-pages:
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-java@v1
+      with:
+        java-version: '11'
+        
+    - uses: actions/setup-go@v1
+      with:
+        go-version: '1.13.x'
+
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.6'
+        architecture: 'x64'
+
+#    - name: Install Groovy
+#      run: sudo apt-get update && sudo apt-get install groovy -y
+
+#    - name: Create ClassPath File
+#      run: mvn dependency:build-classpath -Dmdep.outputFile=target/classpath -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+
+#    - name: Generate Groovy Docs
+#      run: 
```

Comment:
@marcusholl any idea how to squeeze the groovy docs generation in here? 

`createDocu.sh` is [failing](https://github.com/CCFenner/jenkins-library/runs/572111181?check_suite_focus=true)

Reply:
issue was Java11, switched to Java8